source/shared/routes.js

```diff
@@ -0,0 +1,8 @@
+import { Route } from 'react-router';
+import createApp from './components/App';
+
+export default React => () => {
+  const App = createApp(React);
+
+  return <Route path="/" component={App} />;
```

Comment:
The routes aren't returned the expected object for the server side rendering.
When I remove the factory like this:

```
import React from 'react';
import { Route } from 'react-router';
import createApp from './components/App';

const App = createApp(React);

export default (
  <Route path="/" component={App} />
);
```

I get a `Cannot call class as a function` error.

Changed `import createRoutes from 'shared/routes';` to `import routes from 'shared/routes';` as well.

Everything runs fine without errors, but the `App` component isn't being loaded. It shows a `Not found` message (see `server/routes/main/index.js`)